packages/truffle-compile/compilerSupplier/loadingStrategies/LoadingStrategy.js

```diff
@@ -15,11 +15,15 @@ class LoadingStrategy {
     };
     this.config = Object.assign({}, defaultConfig, options);
 
-    const compilerCachePath = path.resolve(
+    let compilerCachePath = path.resolve(
       Config.getTruffleDataDirectory(),
       "compilers"
     );
-    if (!fs.existsSync(compilerCachePath)) fs.mkdirSync(compilerCachePath);
+    if (!fs.existsSync(compilerCachePath)) {
+      fs.mkdirSync(compilerCachePath);
+      compilerCachePath += "/node_modules"; // because babel binds to require & does weird things
```

Comment:
Use the `path` module for this so that it will also work on Windows machines.

Reply:
Good catch!